packages/blockchain-wallet-v4-frontend/src/scenes/Home/BalancesChart/template.success.js

```diff
@@ -86,10 +86,10 @@ const BalancesChart = (props) => {
             <FormattedMessage id='scenes.home.balanceschart.bch' defaultMessage='Bitcoin Cash' />
           </Text>
           <CoinBalance onClick={handleCoinDisplay}>
-            <SwitchableDisplay coin='BCH' size='14px' weight={200}>{bchBalance}</SwitchableDisplay>
+            <SwitchableDisplay coin='BCH' cursor='pointer' size='14px' weight={200}>{bchBalance}</SwitchableDisplay>
```

Comment:
Cant we just do cursor: pointer in one or two places (SwitchableDisplay, FiatDisplay, etc) instead of inline?

Reply:
I thought about that but I don't think all SwitchableDisplays and FiatDisplays should have cursor: pointer

Reply:
Yes you're right, we should do that but only for SwitchableDisplay